letsencrypt_s3front/authenticator.py

```diff
@@ -74,6 +74,7 @@ def cleanup(self, achalls):
         # pylint: disable=missing-docstring,no-self-use,unused-argument
         s3 = boto3.resource('s3', region_name=self.conf('s3-region'))
         client = s3.meta.client
-        for achall in achalls:
-            client.delete_object(Bucket=self.conf('s3-bucket'), Key=achall.chall.path[1:])
+        if achalls is not None:
```

Comment:
Were you actually getting something `None`-y here? This should always come from the client as something...